cmake/Finddpdk.cmake

```diff
@@ -136,6 +136,13 @@ if ("bsd" IN_LIST dpdk_PC_LIBRARIES)
   list (APPEND dpdk_dependencies "bsd")
 endif ()
 
+# As of DPDK 23.07, if libarchive-dev is present, it will make DPDK depend on the library.
+# Unfortunately DPDK also has a bug in its .pc file generation and will not include libarchive
+# dependency under any circumstance. Accordingly, the dependency is added explicitly if libarchive
+# exists.
+pkg_check_modules(libarchive_PC libarchive)
```

Comment:
could you pass `QUIET` parameter to this call. as, whether `libarchive` is detected, is not that important here. so would be better to silence the output.

also, could you add a space before `(`. to be more consistent with the rest of this source file?

Reply:
> also, could you add a space before (. to be more consistent with the rest of this source file?

Sorry about this, I copied from a couple of lines below.

Reply:
my bad. i wasn't quite consistent when creating this cmake module.